gammapy/morphology/models.py

```diff
@@ -67,6 +67,29 @@ class Shell2D(Fittable2DModel):
             \\right.
 
     With :math:`r_{out} = r_{in} + \\mathrm{width}`.
+
+
+    Examples
+    --------
+    .. plot::
+        :include-source:
+
+        import numpy as np
+        from gammapy.morphology import Shell2D
+        import matplotlib.pyplot as plt
+        
+        x0, y0 = 25, 25
```

Comment:
You could also position the source off-center, e.g. `(x0, y0) = (30, 20)` to make it easier for users to see if the x and y axis are what they think. But if you like the centered image better, sure, OK.